### PR TITLE
added: products page

### PR DESCRIPTION
**Pull request by rkapoor10**
Added products listing page   
- Navbar
- Filters   
   - Price
   - Category
   - Ratings
   - Sort
   - Discount

- Product Listing Section 
- Footer
prod page

### CODE REVIEW

File: ecstylesheets/base.css [hunk 1]
```diff
@@ -27,4 +27,31 @@
 .fs-07{
     font-size: 0.7rem;
 }
+.seperator{
+    border-bottom: 1px solid #494b4c;
+}
+
+.padtb-1{
```
Comment (Priya-Patel31) [inline ecstylesheets/base.css:34]: use better naming convention like py-1 this represents padding along y-axis as 1rem

File: index.html [hunk 2]
```diff
@@ -105,108 +99,118 @@ <h3 class="fs-2">Apparel</h3>
...
                 </div>
-
             </div>
         </div>
 
-
         <div class="lp-prod-banner">
             <div class="grid-50-layout bg-lightshade-blue">
-                <div>
+                <div class="">
```
Comment (Priya-Patel31) [inline index.html:114]: remove this empty class.

**Review by Priya-Patel31 — APPROVED** [review]
Everything seems good and you can merge it .

File: index.html [hunk 3]
```diff
@@ -33,70 +36,61 @@
...
                         </div>
-
                     </a>
                 </div>
                 <div class="catg-card">
-                    <a href="http://">
+                    <a href="/products-page/products.html">
                         <div class="overlay-grid">
                             <div class="circle center background bg-baseblue">
                                 <!-- <img class="img-responsive" src="https://picsum.photos/200" alt=""> -->
```
Comment (JantuDeb) [inline index.html:74]: remove commennted code

**Review by JantuDeb — APPROVED** [review]
Suggested changes.

**Pull request merged**